pkg/mark/renderer/blockquote.go

```diff
@@ -81,6 +85,15 @@ func ParseBlockQuoteType(node ast.Node, source []byte) BlockQuoteType {
 			if node.Kind() == ast.KindText {
 				n := node.(*ast.Text)
 				t = ClassifyingBlockQuote(string(n.Text(source)))
+				if t == None {
```

Comment:
Can you add a comment here what this part does? (it will help later to remember why this is here)

Reply:
Added comment in code